Playgrounds/1a_I_See_No_Nil.playground/contents.swift

```diff
@@ -146,7 +146,7 @@ For example the same does not apply to other ‘containers’, such as **Array**
 
 Whereas the following is acceptable:
 
-    let x:Int? = 8 // result is: Optional<Int>.Some(8)
+    let x:Int? = 8 // result is: Optional<Int>.some(8)
 
 * * *
 
```

Comment:
It's understandable that they removed the override for the comparison operators with Optionals. But when generics are extended to support conditional compliance (or whatever the term might be?):

`extension Optional: Comparable where Wrapped: Comparable { …`
(syntax might be different?)

The semantics of the comparison will be identical to how they are currently, so I'm not sure what the benefit is of removing the operator override? I don't imagine that Swift will prevent Optionals from conforming to the `Comparable` Protocol when generics allows it, and it'd be odd if it weren't part of the standard library. 
… Anyway, that's just me waffling about the topic : )